### DISCUSSION TITLE
vweb increases memory usage

### DISCUSSION THREAD

**Discussion by renlite** [vlang/v#12978] [General]
I use V version 0.2.4 and Vweb memory usage is increased on every request without reduction later (win10). I tried the flag `-autofree` but no change.
`> v.exe run serv.v -autofree`
Is this an issue?
```v
import vweb

struct App {
    vweb.Context
}

fn main() {
	mut app := &App{}
	app.Context.handle_static('.', true)
	vweb.run(app, 8080)
}

// This endpoint can be accessed via http://localhost:port/hello
fn (mut app App) hello() vweb.Result {
	return app.text('Hello')
}

// This endpoint can be accessed via http://localhost:port/foo
["/foo"]
fn (mut app App) world() vweb.Result {
	return app.text('World')
}
```

**Comment by boonsboos** [comment]
might be very late, but compiler arguments should be passed *before* `run`. anything after will be passed to the run command
